src/components/ListItem.jsx

```diff
@@ -15,22 +17,48 @@ export function ListItem({
 	dateLastPurchased,
 	dateNextPurchased,
 	totalPurchases,
+	refTime,
+	daysToNext,
 }) {
 	const DAYINMS = 86400000;
 	// sync up checked or not checked data from the database to the page upon page refresh
 	const [isPurchased, setIsPurchased] = useState(isChecked);
 
+	const urgency = [
+		['overdue', 'Overdue'],
+		['soon', 'Soon'],
+		['kind-of-soon', 'Kind Of Soon'],
+		['not-soon', 'Not Soon'],
+		['inactive', 'Inactive'],
+	];
+	const [urgencyIndex, setUrgencyIndex] = useState(4); //requires a default value else throws error
```

Comment:
```suggestion
	const [urgencyIndex, setUrgencyIndex] = useState(urgency[4]); //requires a default value else throws error
```

Reply:
Thanks for these and all the major and minor suggestions that ultimately make this more readable!